### DISCUSSION TITLE
Use Laragon test sites as production

### DISCUSSION THREAD

**Discussion by TedMurphy** [leokhoa/laragon#481] [Q&A]
How do I make the .test site available to the world outside as a production site?  I want to use this same VPS server.  It has a domain name assigned to it within DNS.  So, I want to go to myproject.com externally, using the same website as myproject.test internally.

Thank you!!

**Comment by Pen-y-Fan** [comment]
Sorry, I'm not clear on what you are asking.

1. Are you trying to deploy a local app: myproject.test to a production server which will host myproject.com?
2. Are you trying to temporarily give a link from myproject.test so someone external, without needing to deploy it?

For 1. you need a host with LAMP or LEMP stack which you can deploy to. Every host is different, you will need to follow the instructions from the host.

For 2. you need NGOK or similar.

**Comment by TedMurphy** [comment]
I was trying to use one machine for both Laragon development and production.  I've tried several solutions, but I think your answer has nailed it.  Thank you very much for the response.

**Reply by Pen-y-Fan** [reply]
Your welcome, Laragon is designed as a local development environment. Your production environment would need to be 'locked down' with only the minimum required to run. The least to keep secure and up to date the better.
